app/bundles/EmailBundle/EventListener/CampaignSubscriber.php

```diff
@@ -73,21 +74,24 @@ class CampaignSubscriber implements EventSubscriberInterface
      * @param EventModel        $eventModel
      * @param MessageQueueModel $messageQueueModel
      * @param SendEmailToUser   $sendEmailToUser
+     * @param MauticFactory     $factory
      */
     public function __construct(
         LeadModel $leadModel,
         EmailModel $emailModel,
         EventModel $eventModel,
         MessageQueueModel $messageQueueModel,
         SendEmailToUser $sendEmailToUser,
-        TranslatorInterface $translator
+        TranslatorInterface $translator,
+        MauticFactory $factory
```

Comment:
MauticFactory is deprecated and will remove in 3.0
Use service, in your case CoreParametersHelper

Reply:
@kuzmany thanks for the feedback!
I changed the code, please review it again.